src/include/sof/lib/pm_runtime.h

```diff
@@ -47,8 +47,30 @@ enum pm_runtime_context {
 struct pm_runtime_data {
 	spinlock_t lock;	/**< lock mechanism */
 	void *platform_data;	/**< platform specific data */
+#if CONFIG_POWER_STATE_COUNTERS
+	struct p_counters_data *p_counters; /**< debug power state counters */
+#endif
 };
 
+#if CONFIG_POWER_STATE_COUNTERS
+/**
+ * \brief Power states
```

Comment:
Same comment as for the Kconfig change. Subst "power state" with "dsp residency".